sapi/cli/php_cli_server.c

```diff
@@ -2421,7 +2421,7 @@ static void php_cli_server_startup_workers(void) {
 	if (php_cli_server_workers_max > 1) {
 		zend_long php_cli_server_worker;
 
-		php_cli_server_workers = calloc(
+		php_cli_server_workers = ecalloc(
```

Comment:
`ecalloc()` and friends allocate memory which is [freed at the end of each *request*](https://www.phpinternalsbook.com/php7/memory_management/zend_memory_manager.html). That would cause issues here. Instead you'd need `pecalloc()` with the third argument being `1` (and use `pefree()` instead of `efree()` above). Note that the Zend memory allocation functions are infallible, i.e. they never return `NULL` (but instead terminate the process), so the following `NULL` check is superfluous.